### PR TITLE
Applying apache license and adding license header on source files

### PR DESCRIPTION
**Pull request by s26mehta**
All the builds on wercker build without error

### CODE REVIEW

**Comment by bcwaldon** [comment]
@s26mehta we don't want to touch any of the vendored code (anything with vendor/ in the path)

**Comment by s26mehta** [comment]
@bcwaldon okay I have unedited vendor code.

File: group_vars/all
Comment (bcwaldon) [inline group_vars/all]: added an `f` to the end of this line, is that necessary?

**Review by bcwaldon — CHANGES_REQUESTED** [review]
Please revert the changes to the files in `roles/defunctzombie.coreos-bootstrap/`, as this is not planet code.

**Comment by s26mehta** [comment]
Fixed both

**Review by bcwaldon — APPROVED** [review]

**Comment by bcwaldon** [comment]
Made the remaining changes. Waiting for wercker sign-off

**Pull request merged**
